ports/stm/boards/swan_r5/board.c

```diff
@@ -49,6 +65,7 @@ void initialize_discharge_pin(void) {
     GPIO_InitStruct.Pin = GPIO_PIN_4;
     HAL_GPIO_Init(GPIOE, &GPIO_InitStruct);
     HAL_GPIO_WritePin(GPIOE, GPIO_PIN_4, GPIO_PIN_SET);
```

Comment:
Not going to block this PR *but* you could use the common_hal APIs here to set the pin value. That'd be simpler and ensure that any future DigitalInOut state is updated. (It doesn't look like any state is stored currently.) Feel free to open another PR for it.

Reply:
Thanks - I did wonder about that, but like you, I checked that all the state was coming from the peripherals themselves. +1 to using the common hal APIs being cleaner and won't cause any surprises with future mods to DigitalInOut state.